Gemfile

```diff
@@ -1,14 +1,7 @@
-ENV['rails'] ||= '4.2.0.rc2'
+ENV['rails'] ||= '4.2.0'
 
 source 'https://rubygems.org'
 
 gem 'rails', "~> #{ENV['rails']}"
 
-if ENV['rails'][0] == '4'
-  gem 'database_cleaner'
-end
-if ENV['rails'] =~ /4.0|3.2/
-  gem 'rubysl-test-unit'
-end
```

Comment:
Can we remove this also from https://github.com/doorkeeper-gem/doorkeeper/blob/master/gemfiles/Gemfile.common.rb#L7-L12?